Source/Mapping/Mapping.csproj

```diff
@@ -9,9 +9,9 @@
     <Company>Affecto</Company>
     <PackageProjectUrl>https://github.com/affecto/dotnet-Mapping</PackageProjectUrl>
     <PackageTags>affecto map mapping</PackageTags>
-    <PackageReleaseNotes>Converted to .NET Standard 2.0</PackageReleaseNotes>
+    <PackageReleaseNotes>Added possibility to pass parameters to a mapping profile</PackageReleaseNotes>
```

Comment:
"Added possibility to pass parameters to a mapping profile. Deprecated two-way-mapping."